scripts/run_mission1_robustness.py

```
@@ -0,0 +1,519 @@
+﻿#!/usr/bin/env python3
```

Ruff (0.15.9)

[warning] Shebang should be at the beginning of the file

(EXE005)